pom.xml

```diff
@@ -43,6 +43,7 @@
         <jcommander.version>1.35</jcommander.version>
         <log4j.version>1.2.17</log4j.version>
         <jackson.version>2.9.9</jackson.version>
+        <jackson-databind.version>2.9.9.1</jackson-databind.version>
```

Comment:
Why have a discrepancy between `jackson.version` and `jackson-databind.version`?

Reply:
2.9.9.1 and 2.9.9.2 are "micro-patches" that were only released for `jackson-databind`, see https://github.com/FasterXML/jackson/wiki/Jackson-Release-2.9#micro-patches